yarn.lock

```diff
@@ -11838,7 +11838,7 @@ preact@^10.5.13:
   resolved "https://registry.yarnpkg.com/preact/-/preact-10.13.2.tgz#2c40c73d57248b57234c4ae6cd9ab9d8186ebc0a"
   integrity sha512-q44QFLhOhty2Bd0Y46fnYW0gD/cbVM9dUVtNTDKPcdXSMA7jfY+Jpd6rk3GB0lcQss0z5s/6CmVP0Z/hV+g6pw==
 
-prebid.js@guardian/prebid.js#91cabf5cdfc3c01745f4b10e55044f95a9d7d1b4:
+"prebid.js@github:guardian/prebid.js#91cabf5cdfc3c01745f4b10e55044f95a9d7d1b4":
```

Comment:
Can we remove the changes generated on this line? This is a known issue.

Reply:
Thank you. I just ran `yarn cache clean` and `yarn install -fix" which should resolve this.